### PR TITLE
SLE 12/15 CCE/CIS codes added and CIS codes updated

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _Assignment of CCE/CIS codes or CIS codes updates _

#### Rationale:

- Some of the SLE 12/15' rules in the CIS group 4.Logging and Auditing did not have CCE/CIS codes assigned 

#### Review Hints:

- _Review hints here. Replace this text. Don't use the italics format!_

- _Use this optional section to give any relevant information which could help the reviewer to more quickly and assertively understand and test the changes._

- _Good examples are useful commands, if it is better to review all commits together or in a suggested sequence, any relevant discussion in other PRs or issues, etc._

### CODE REVIEW

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rumen

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rumen

File: linux_os/guide/system/logging/rsyslog_accepting_remote_messages/rsyslog_nolisten/rule.yml [hunk 1]
```diff
@@ -27,11 +27,14 @@ identifiers:
...
     cce@rhel8: CCE-84275-7
     cce@rhel9: CCE-83995-1
+    cce@sle12: CCE-92259-1
+    cce@sle15: CCE-91375-6
 
 references:
     cis-csc: 1,11,12,13,14,15,16,18,3,4,5,6,8,9
     cis@alinux3: 4.2.1.6
-    cis@rhel7: 4.2.1.5
+    cis@rhel7: 4.2.1.
```
Comment (marcusburghardt) [inline linux_os/guide/system/logging/rsyslog_accepting_remote_messages/rsyslog_nolisten/rule.yml:36]: Is it a typo or is the id incomplete?

**Review by marcusburghardt — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/logging/rsyslog_accepting_remote_messages/rsyslog_nolisten/rule.yml (hunk 1 above)
Comment (rumch-se) [reply linux_os/guide/system/logging/rsyslog_accepting_remote_messages/rsyslog_nolisten/rule.yml]: Hello @marcusburghardt 
It is a typo. I have done the expected correction.
Have a nice day
Rumen

**Review by marcusburghardt — APPROVED** [review]

**Pull request merged**